views/forms/UserProfileForm/UserProfileFormFields.css

```diff
@@ -0,0 +1,3 @@
+.UserProfileFormInputs_box {
+  padding-right: 506px;
```

Comment:
why do we need this?

Reply:
The inputs are either full width or like 10% of full width, so I had to wrap them in a div and make sure the div takes up the correct space as in the design without having to explicitly set the width

Reply:
we should be able to have them be full width and put a max-width constraint on them so they don't exceed a certain width.